### ISSUE TITLE
Doesn't do anything?

### ISSUE THREAD

**Issue by arnlen** [pommiegranit/mailchimp-webhooks#1]
With Version 4.1.1, when I trigger an event from MailChimp, nothing happens in Wordpress.
I can't check the log file because it dosn't exist.

When I manually try with Postman, the answer from the server is empty.

**Comment by pommiegranit** [comment]
Arnlen, re-tested with 4.1.1 and works as expected. If you are not getting anything at all then it's probably the WP URL that you are using. 

I did notice an error with the webhook URL that is output in the Settings page (I've updated the code to correct this). Your webhook URL should be:

http://[your site domain]/webhook/?key=yoursecretkey

Also, you must be using **pretty permalinks**.

If you call this URL directly (i.e. just put it in your browser location) then it should return the message:

No request details found.

This is a simple way to test that the URL is up and running. 

Let me know how you go.
